ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env

```diff
@@ -11,4 +11,17 @@ authorization_token={{ discussionsmw_nodebb_authorization_token }}
 TELEMETRY_EVENTS_BATCH_SIZE={{ telemetry_events_batch_size | default(1) }}
 TELEMETRY_SERVICE_API_SLUG={{ telemetry_service_api_slug | default('/v1/telemetry') }}
 TELEMETRY_SERVICE_URL={{ telemetry_service_url | default('http://telemetry-service:9001') }}
-API_AUTH_TOKEN={{ api_auth_token | default('') }}
\ No newline at end of file
+API_AUTH_TOKEN={{ api_auth_token | default('') }}
+moderation_flag={{ moderation_flag |  default(false) }}
+moderation_type={{ moderation_type |  default('pre-moderation') }}
+moderation_id={{ moderation_id |  default('GEbX4X0B9pbA_yqYBUtM') }}
+moderation_flag_by={{ moderation_flag_by |  default('system_flagged') }}
+moderation_platform={{ moderation_platform |  default('Sunbird') }}
+moderation_category={{ moderation_category |  default('discussions') }}
+moderation_contentId={{ moderation_contentId |  default('') }}
+moderation_email={{ moderation_email |  default('') }}
+moderation_email_subject={{ moderation_email_subject |  default('Sunbird Discussion  Moderation') }}
+moderation_course={{ moderation_course |  default('Sunbird training') }}
+LEARNER_SERVICE_URL={{ learner_service_url |  default('') }}
+SB_API_KEY={{ SB_API_KEY |  default('') }}
```

Comment:
Which all APIs access is required for this change? We may need to generate the key, so I would like to know the list of apis required.